Source/ACE/Managers/LandblockManager.cs

```diff
@@ -87,6 +86,8 @@ private static Landblock GetLandblock(LandblockId landblockId, bool propogate)
                     {
                         // load up this landblock
                         Landblock block = new Landblock(landblockId);
+                        block.Load();

```

Comment:
By having a mandatory function call after the constructor, you leave open the possibility for the Landblock to be in an invalid state: instantiated, but not loaded.  The design of having the constructor do the Loading work was very intentional and should remain that way.  As best I can tell, you broke this design paradigm for the simple purpose of artificially creating more states.  Please revert the constructor changes.  If that means you lose a state, fine, it's a state that shouldn't exist.